### ISSUE TITLE
Custom CA certificate during connection to vault

### ISSUE THREAD

**Issue by r0bj** [IBM/argocd-vault-plugin#52]
**Is your feature request related to a problem? Please describe.**
Vault server in my organization uses self-signed certificate to serve TLS traffic. Without possibility to configure CA certificate plugin execution generates error:
```
Error: Put "https://vault:8200/v1/auth/approle/login": x509: certificate signed by unknown authority
```

**Describe the solution you'd like**
Add option to provide and configure CA certificate for argocd-vault-plugin for vault connections.

**Comment by clcchai** [comment]
hi is there a way to bypass the cert and ignore cert check?

Unable to create application: application spec is invalid: InvalidSpecError: Unable to generate manifests in .: rpc error: code = Unknown desc = `argocd-vault-plugin generate ./` failed exit status 1: Error: Put "https://vault/v1/auth/approle/login": x509: certificate signed by unknown authority

**Comment by jkayani** [comment]
Hi, this seems like a reasonable requirement - we're looking into how we can do this. Stay tuned!

**Comment by werne2j** [comment]
Code to support this request has been merged into Main and will be released with version 0.5.0

**Comment by werne2j** [comment]
Custom CACERT and CAPATH support available in https://github.com/IBM/argocd-vault-plugin/releases/tag/v0.5.0
